src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('th');
+const tBody = document.querySelector('tBody');
+const rows = document.querySelectorAll('tBody tr');
+
+const array = [...rows];
+
+headers.forEach((header) => {
+  header.onclick = function (e) {
+    const index = e.target.cellIndex;
+
+    sortBy(array, index);
+  };
```

Comment:
It's a good practice to use `addEventListener` instead of `onclick`. Using `addEventListener` allows you to add multiple event handlers to a single event, whereas `onclick` only allows one.